Godeps/Godeps.json

```diff
@@ -305,45 +343,45 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/image-spec/identity",
-			"Comment": "v1.0.0-rc6",
-			"Rev": "1a6593ab6c3ab5902072b4694a22ff19425396ae"
+			"Comment": "v1.0.0-rc6-12-g372ad78",
+			"Rev": "372ad780f63454fbbbbcc7cf80e5b90245c13e13"
```

Comment:
@abhinandanpb Thanks! We need to update containerd version to include this in another PR. It seems that some api changes are included https://github.com/containerd/containerd/pull/1268.